code/modules/mob/living/carbon/carbon.dm

```diff
@@ -215,8 +210,8 @@
 			AdjustKnockdown(-3)
 			playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 			M.visible_message( \
-				"<span class='notice'>[M] shakes [src] trying to wake [t_him] up!</span>", \
-				"<span class='notice'>You shake [src] trying to wake [t_him] up!</span>", \
+				"<span class='notice'>[M] shakes [src] trying to wake \him up!</span>", \
```

Comment:
Not in the repo's history, which is the main one that matters. And the problem is that we shouldn't be at risk of losing unrelated fixes when we do a revert of major changes. If this PR turns out to break something major and we have to go back in an emergency, these changes disappear as well, for no reason. Atomicity is a useful rule for making reverts easy and clean.